extensions/arc/images/gear.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Where were these svg icons sourced from?

Reply:
Copied the SVG off of figma, https://www.figma.com/file/fQO2yNBwr773QI4ANvb1Z4/Azure-Portal-Icons?node-id=2626%3A0